make run fails

```
$ make run
go generate ./pkg/... ./cmd/...
go fmt ./pkg/... ./cmd/...
go vet ./pkg/... ./cmd/...
go run ./cmd/manager/main.go
# github.com/maestrosdk/maestro/vendor/k8s.io/client-go/transport
vendor/k8s.io/client-go/transport/round_trippers.go:437:9: undefined: strings.Builder
# github.com/maestrosdk/maestro/vendor/sigs.k8s.io/kustomize/pkg/target
vendor/sigs.k8s.io/kustomize/pkg/target/kusttarget.go:89:5: dec.DisallowUnknownFields undefined (type *json.Decoder has no field or method DisallowUnknownFields)
make: *** [run] Error 2
```

--- a/README.md
+++ b/README.md
@@ -2,9 +2,17 @@
 
 Maestro provides a declarative approach to building production-grade Kubernetes Operators covering the entire application lifecycle. 
 
+## Pre-requisites
+
+Before you get started:
+
+- Install Go `1.11` or later
+- Latest version of `dep`
+- Kubernetes Cluster `1.12` or later (e.g. [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/))
+- [Configure kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+
 ## Installation Instructions
 
-- [Configure kubectl ](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
 - Get maestro repo: `go get github.com/maestrosdk/maestro/`
 - `cd $GOPATH/src/github.com/maestrosdk/maestro`
 - `make install` to deploy universal CRDs

